src/main/java/com/acoustic/salarycalculator/SalaryCalculatorApplication.java

```diff
@@ -1,19 +1,19 @@
-package org.example.enumApproch;
+package com.acoustic.salarycalculator;
 
 import java.math.BigDecimal;
 
-import org.example.enumApproch.menu.MenuSalaryCalculator;
-import org.example.enumApproch.print.PrinterSalaryCalculator;
-import org.example.enumApproch.service.JobTitleService;
-import org.example.enumApproch.service.JobTitleServiceImpl;
+import com.acoustic.salarycalculator.menu.SalaryCalculatorMenu;
+import com.acoustic.salarycalculator.printer.SalaryCalculatorPrinter;
+import com.acoustic.salarycalculator.service.JobTitleService;
+import com.acoustic.salarycalculator.service.JobTitleServiceImpl;
 
 
-public class MainWithEnum {
+public class SalaryCalculatorApplication {
 
     public static void main(String[] args) {
 
-        MenuSalaryCalculator menu = new MenuSalaryCalculator();
```

Comment:
we don't need to save salary without job title, just save salary with job title by one db request, if job title is not defined you can save with default value, or not save at all

Reply:
you could also extract some code to smaller methods at least 2, one for salary calculcation and second for statistics

Reply:
fixed